README.md

```diff
@@ -38,89 +38,163 @@ You should see output similar to the following:
+
+## create either a development or production image from the same Dockerfile, using commands like:
+$ docker build --target development --tag todo-app:dev .
+$ docker build --target production --tag todo-app:prod .
+# Builds the todo app image
+# Open the docker hub
+ # Click on image
+# Todo app image is created
+ ======================================   
+ Run the container
+ ---------------------
+# Below will create and start the container in dev/prod environment 
+# Open the docker hub
+# Click on Containers/Apps
+# New container will be started 
+ $ docker run --env-file ./.env -p 5100:5000 --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app todo-app:dev
+ $ docker run --env-file ./.env -p 5100:5000 --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app todo-app:prod
+=============================================
+## test the local development setup using a command
+##docker command to build and run Todo App in development environment
+
+$ docker build --target development --tag todo-app:dev .
+$ docker run --env-file ./.env -p 5100:5000 --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app todo-app:dev
+ * Serving Flask app "todo_app/app" (lazy loading)
+ * Environment: development
+ * Debug mode: on
+ * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
+ * Restarting with stat
+ * Debugger is active!
+ * Debugger PIN: 145-903-278
 
-Trouble shooting
-delete .venv
-Edit poetry.toml file
-[virtualenvs]
-create = true
-in-project = false
+Browse to http and change the port to 5100
+http://localhost:5100/
+ToDoApp is displayed
 
-Write a trigger to install dependencies
+=======================================================================================
 
-Add Network to vagrant file
-config.vm.network "forwarded_port", guest: 5000, host: 5000
+# Will build production environment
+FROM builder-base as production
+RUN poetry install --no-dev   
+COPY  ./todo_app  /app/todo_app
+ENTRYPOINT [ "poetry", "run", "gunicorn", "--bind", "0.0.0.0:5000", "todo_app.wsgi:app" ]
+ # docker command to build and run Todo App in Production environment
+$ docker build --target production --tag todo-app:prod .
 
-launch the app after vagrant up
+docker run --env-file ./.env -p 5100:5000 --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app todo-app:prod
+ * Serving Flask app "todo_app/app" (lazy loading)
+ * Environment: development
+ * Debug mode: on
+ * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
+ * Restarting with stat
+ * Debugger is active!
+ * Debugger PIN: 145-903-278
+ # Browse to http and change the port to 5100 on browser to view todoapp
+http://localhost:5100/
+# ToDoApp is displayed in prod environmnet
+===================================
+## To view the container logs
+docker logs <CONTAINER ID>
+=======================================
+## Docker Compose
+launch your containers in a single yaml file (docker-compose.yml)
+to launch use a single command (docker-compose up) 
```

Comment:
Except your docker-compose file isn't written yet - it just contains some placeholders.

Reply:
ok